community/ndp/parser-v1/src/main/antlr4/org/neo4j/tankparse/TankV1.g4

```diff
@@ -0,0 +1,84 @@
+/** Temporary pre-parser that handles "tank"-codenamed cypher extensions.
+    Functionality here will be consolidated with the regular cypher parser. */
+
+grammar TankV1;
```

Comment:
Using antlr was quite intentional, an experiment to try it out. It's got some nice features - *very* fast, minimal GC overhead (it does not build an AST object graph, uses visitors during the parsing instead, *very* useful to handle large statements), and of course the grammar is much more familiar to non-scala peeps.

But it was just that, an experiment. I'll drop the parser out of this PR, there's no need for it now anyway since we moved the u-exts out.